pkg/security/ebpf/c/erpc.h

```diff
@@ -55,15 +57,16 @@ int __attribute__((always_inline)) handle_discard_pid(void *data) {
 }
 
 int __attribute__((always_inline)) is_eprc_request(struct pt_regs *ctx) {
-    u64 fd, pid;
+//    u64 fd, pid;
+    u64 pid;
 
-    LOAD_CONSTANT("erpc_fd", fd);
+//    LOAD_CONSTANT("erpc_fd", fd);
     LOAD_CONSTANT("erpc_pid", pid);
 
-    u32 vfs_fd = PT_REGS_PARM2(ctx);
-    if (!vfs_fd || (u64)vfs_fd != fd) {
-        return 0;
-    }
+//    u32 vfs_fd = PT_REGS_PARM2(ctx);
```

Comment:
Why removing this check ?

Reply:
The eRPC clients were initialized at 2 different places in the code. Updated and fixed ;)